deployer/src/algolia_internal_api.py

```diff
@@ -0,0 +1,122 @@
+import requests
+from os import environ
+from base64 import b64encode
+
+
+def get_endpoint(endpoint, params=''):
+    base_endpoint = environ.get('BASE_INTERNAL_ENDPOINT')
+
+    return base_endpoint + endpoint + params
+
+
+def get_headers():
+    token = environ.get('INTERNAL_API_AUTH')
+
+    app_id = environ.get('APPLICATION_ID_PROD')
+    admin_api_key = environ.get('API_KEY_PROD')
+    auth_token = b64encode(app_id + ":" + admin_api_key).replace('=', '').replace("\n", '')
+
+    return {
+        'Authorization': 'Basic ' + token,
+        'Algolia-Application-Authorization': 'Basic ' + auth_token
+    }
+
+
+def get_application_rights():
+    app_id = environ.get('APPLICATION_ID_PROD')
+    endpoint = get_endpoint('/applications/' + app_id)#, '?fields=application_rights')
+
+    r = requests.get(endpoint, headers=get_headers())
+
+    data = r.json()
+
+    return data['application_rights']
+
+
+def get_right_for_email(email):
+    rights = get_application_rights()
+
+    for right in rights:
+        if right['user']['email'] == email:
+            return right
+
+
+def get_indices_for_right(right):
+    if right is not None:
+        return right['indices']
+    return []
+
+
+
+def add_user_to_index(index_name, user_email):
```

Comment:
I updated this method to return different values based on the outcome of the call. `True` when adding an existing user, `None` when already added and the url to the invitation url if the user does not yet have an Algolia account. This will in turn be used by the snippeter to chose what to add to the note